hw_9_1.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<p>It is missing what you actually want to do with the data</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>